resources/conf/runners/Library.py

```diff
@@ -0,0 +1,125 @@
+from browserstack.local import Local
+import json
+import csv
+import os
+from robot.libraries.BuiltIn import BuiltIn
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.keys import Keys
+
+
+bstack_local = 0
+se2lib = BuiltIn().get_library_instance('SeleniumLibrary')
+
+def combine_dict(dict1, dict2):
+    dict_1 = json.loads(str(dict1))
+    dict_2 = json.loads(str(dict2))
+
+    dict_1.update(dict_2)
+
+    return dict_1
+
+
+def get_caps_single(caps_path, username, access):
+
+    with open(caps_path) as f:
+        data = json.load(f)
+
+    single_caps_common = data['tests']['single']['common_caps']
+    single_caps_env = data['tests']['single']['env_caps'][0]
+
+    single_caps_common.update(single_caps_env)
+    single_caps_common['browserstack.user'] = username
+    single_caps_common['browserstack.key'] = access
+
+    return single_caps_common
+
+def get_caps_local(caps_path, username,access):
```

Comment:
are you using a formatter?

def get_caps_single(caps_path, username, access):
def get_caps_local(caps_path, username,access):

a formatter will take care of the space post the comma. and the code will also look more neater when it follows the right formatting guidelines.

Reply:
run the formatter across all files.